Extensions/InternetConnectivity.json

```diff
@@ -20,19 +25,38 @@
   ],
   "dependencies": [],
   "eventsFunctions": [
+    {
+      "description": "",
+      "fullName": "",
+      "functionType": "Action",
+      "group": "",
+      "name": "onFirstSceneLoaded",
+      "private": false,
+      "sentence": "",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": "  window._extensionInternetStatusChecker = {\r\n    Status: false,\r\n    UrlBasis: 'https://example.com/',\r\n    FrequencyCheckSeconds: 5\r\n  };\r\n\r\n\r\n  // Checks internet via ping method\r\n  const checkOnlineStatus = async () => {\r\n    try {\r\n      const online = await fetch(window._extensionInternetStatusChecker.UrlBasis, {\r\n        method: 'GET',\r\n        mode: 'no-cors',\r\n        cache: 'no-cache',\r\n      });\r\n\r\n      return online.status == 0;\r\n    } catch (e) {\r\n      return false;\r\n    }\r\n  };\r\n\r\n  // Check internet if browser/device detects an event\r\n  window.addEventListener('load', detectInternet);\r\n  window.addEventListener('online', detectInternet);\r\n  window.addEventListener('offline', detectInternet);\r\n  setInterval(detectInternet, window._extensionInternetStatusChecker.FrequencyCheckSeconds*1000);\r\n\r\n  // Update Status in a variable\r\n  async function detectInternet() {\r\n    if(await checkOnlineStatus()) {\r\n        window._extensionInternetStatusChecker.Status = true;\r\n    } else {\r\n        window._extensionInternetStatusChecker.Status = false;\r\n    }\r\n  }\r\n\r\n  detectInternet();",
```

Comment:
- Avoid storing things on window, rather store things on `gdjs`.
- The status check is made every 5 seconds, could this be customised? Otherwise this can create a lot of network calls.
- What happen if example.com is not available?
- nitpicking: Use `camelCase` for properties, not `PascalCase`

Reply:
> Avoid storing things on window, rather store things on gdjs.
I cant somehow the builder wont let me. I used the documentation convention see this one https://github.com/GDevelopApp/GDevelop-extensions/issues/463 builder logs

Reply:
> The status check is made every 5 seconds, could this be customised? Otherwise this can create a lot of network calls.

Hmm I could add an action where users have to call it initial e.g. "ping this website _PARAM_ every _PARAM_ seconds" 

> What happen if example.com is not available?

don't know any website that is available to all countries even google.com is banned from china, so not sure what to put there

Reply:
> nitpicking: Use camelCase for properties, not PascalCase

Updated